ocaml/testsuite/tests/typing-layouts-err-msg/probe.reference

```diff
@@ -0,0 +1,10 @@
+File "probe.ml", line 12, characters 6-7:
+12 |   let f () = x in
+           ^
+Warning 26 [unused-var]: unused variable f.
+File "probe.ml", lines 11-14, characters 20-2:
+11 | ....................[%probe "a" (
+12 |   let f () = x in
+13 |   ()
+14 | )]
+Error: Variables in probe handlers must have layout value, but x in this handler does not.
```

Comment:
This is very sad, as the comments in translcore describe. Just update the layout-histories text to tell a reader to call for help.